packages/npm-package-json-lint-config/index.js

```diff
@@ -95,6 +95,7 @@ const defaultConfig = {
 		'require-private': 'off',
 		'require-publishConfig': 'off',
 		'require-repository': 'error',
+		'require-repository-directory': 'error',
```

Comment:
We do *not* want this in the _package_ distributed configuration, the goal of this package is to have it used by any project, not just Gutenberg.

The majority of projects that will use the `@wordpress/npm-package-json-lint-config` package do not use Lerna.

As such, the above change should be moved from the above to the _root_ `package.json` configuration here https://github.com/WordPress/gutenberg/blob/master/package.json#L124-L150